config/feeds_config_v2.json

```diff
@@ -52880,7 +52880,68 @@
         "decimals": 0,
         "category": "",
         "market_hours": "Crypto",
-        "arguments": []
+        "arguments": [],
+        "rpc_urls": ["https://eth.llamarpc.com", "https://ethereum-rpc.publicnode.com", "https://rpc.eth.gateway.fm"]
+      }
+    },
+    {
+      "id": 100002,
+      "full_name": "YToken exchangeRate",
+      "description": "YToken exchangeRate call on 0x19Ebd191f7A24ECE672ba13A302212b5eF7F35cb",
+      "type": "price-feed",
+      "oracle_id": "eth-rpc",
+      "value_type": "numerical",
+      "stride": 0,
+      "quorum": {
+        "percentage": 100,
+        "aggregation": "median"
+      },
+      "schedule": {
+        "interval_ms": 90000,
+        "heartbeat_ms": 3600000,
+        "deviation_percentage": 0.1,
+        "first_report_start_unix_time_ms": 0
+      },
+      "additional_feed_info": {
+        "pair": {
+          "base": "",
+          "quote": ""
+        },
+        "decimals": 0,
+        "category": "",
+        "market_hours": "Crypto",
+        "arguments": [],
+        "rpc_urls": ["https://eth.llamarpc.com", "https://ethereum-rpc.publicnode.com", "https://rpc.eth.gateway.fm"]
```

Comment:
If you still decide to go with this approach, make sure to update the feed config schema in `libs/ts/config-types/src/data-feeds-config`. Otherwise, we risk losing the `rpc_urls` field the next time we auto-generate `config/feeds_config_v2.json`.

Reply:
I understand, but this could lead to bigger issues down the line. Whenever we make changes to the oracles (and, by extension, the feeds config), we need to ensure the schemas are kept in sync. Otherwise, we risk inconsistencies that are hard to track and debug later.

I added a commit to your branch that defines the `arguments` for the `eth-rps` oracle. The build should now pass.